react_frontend/src/components/App.js

```diff
@@ -1,84 +1,102 @@
 import React, { Component } from 'react'
-import Table from './Table'
-import Form from './Form'
-import axios from 'axios'
+import NavBar from './NavBar'
+import Spreadsheet from './Spreadsheet'
+import Home from './Home'
+import {BrowserRouter as Router, Switch, Route} from 'react-router-dom'; 
+
 
 class App extends Component {
-  state = {
-    characters: [],
+  render() {
+    // const { characters } = this.state
+    return(
+      <Router>
+        <div className="App">
+          <NavBar />
+          <Switch>
+            <Route exact path="/Spreadsheet" component={Spreadsheet}/> 
+            <Route exact path="/" component={Home}/> 
+          </Switch>
+        </div>
+      </Router>
+    )
   }
-  removeCharacter = index => {
-    const { characters } = this.state
+}
 
-    //this.makeDeleteCall(characters[index])
-    axios.delete('http://localhost:5000/users', {data : characters[index]})
+//   state = {
+//     characters: [],
+//   }
+//   removeCharacter = index => {
+//     const { characters } = this.state
 
+//     //this.makeDeleteCall(characters[index])
+//     axios.delete('http://localhost:5000/users', {data : characters[index]})
```

Comment:
Might want to delete these comments as they seem to be jumbled old code.

Reply:
Ya, will get to it!